src/curated-corpus/pages/CorpusItemPage/CorpusItemPage.tsx

```diff
@@ -107,26 +109,26 @@ export const CorpusItemPage: React.FC = (): JSX.Element => {
               </Box>
 
               <ApprovedItemCurationHistory
-                item={data.approvedCorpusItemByExternalId!}
+                item={data.approvedCorpusItemByExternalId! as any}
               />
             </Grid>
           </Grid>
           <RejectCorpusItemAction
-            item={data.approvedCorpusItemByExternalId!}
+            item={data.approvedCorpusItemByExternalId! as any}
             actionScreen={ActionScreen.Corpus}
             modalOpen={rejectModalOpen}
             toggleModal={toggleRejectModal}
             refetch={refetch}
           />
           <ScheduleCorpusItemAction
-            item={data.approvedCorpusItemByExternalId!}
+            item={data.approvedCorpusItemByExternalId! as any}
             modalOpen={scheduleModalOpen}
             toggleModal={toggleScheduleModal}
             refetch={refetch}
             actionScreen={ActionScreen.Corpus}
           />
           <EditCorpusItemAction
-            item={data.approvedCorpusItemByExternalId!}
+            item={data.approvedCorpusItemByExternalId! as any}
```

Comment:
I think you can solve for these type mismatches by running `npm run api:generate-types`.

Let's stay away from `as any` unless absolutely necessary, because it can hide incompatibilities.